src/ui/components/modals/AppModal.tsx

```diff
@@ -72,6 +73,7 @@ export const AppModal: FunctionComponent<Props> = ({
   maxHeight,
   shouldScrollToEnd,
   keyboardShouldPersistTaps,
+  shouldRemoveSpacerBetweenHeaderAndContent,
```

Comment:
```suggestion
  addSpacerBetweenHeaderAndContent = true,
```

Je n'aime pas tellement avoir des conditions négatives. C'est une mauvaise pratique et ça rend le code beaucoup plus difficile à lire.
Il vaut donc mieux appeler `<Comp addSpacerBetweenHeaderAndContent={false} />`

Reply:
J'avais choisi de faire comme ça car ça serait le seul cas sans séparateur mais je peux inverser en forçant une valeur par défaut à true du coup